main/fopen_wrappers.c

```diff
@@ -110,6 +110,15 @@ PHPAPI ZEND_INI_MH(OnUpdateBaseDir)
 			*end = '\0';
 			end++;
 		}
+#ifndef PHP_WIN32
+		if (ptr[0] == '.' && ptr[1] == '.' && (ptr[2] == '\0' || ptr[2] == DEFAULT_SLASH)) {
+#else
+		if (ptr[0] == '.' && ptr[1] == '.' && (ptr[2] == '\0' || ptr[2] == DEFAULT_SLASH || ptr[2] == '/')) {
```

Comment:
There's an IS_SLASH macro that does this.

Reply:
Oh, right. Thanks!